### DISCUSSION TITLE
We have requirement of adding a risk register, and for that, is there any plugin available

### DISCUSSION THREAD

**Discussion by arunqa** [nautobot/nautobot#1974] [Plugin (App) Ideas & Development]
We are evaluating the Nautobot services for our use cases in a locally installed instance, and we have customized the UI using the custom fields for our use cases. Now we have a requirement of adding a risk register. And for that, is there any available plugin to add the same or any other option like Creating a new panel/menu? Please let me know the options to add a Risk Register in Nautobot. Thanks.

![risk](https://user-images.githubusercontent.com/5555183/175285733-859fc547-21f0-4c12-ba79-48de153bad3c.png)
u

**Comment by bryanculver** [comment]
@arunqa Moved this over to a discussion.

I am unaware at this time if there is a plugin available for such functionality, nor any existing core functionality to support this.

You may also reach out to [our Slack channel](https://github.com/nautobot/community#come-join-us-on-slack) to discuss there.
